src/context.rs

```diff
@@ -171,6 +171,10 @@ impl<'a> Context<'a> {
             _ => Shell::Unknown,
         }
     }
+
+    pub fn elapsed(&self) -> Option<u128> {
```

Comment:
Can we call this something like `get_cmd_duration`?

As a local variable, it was fine for it to have a more ambiguous name since that would only be used within the function. As a method on a struct that's touched by almost every single other module, I'd like something a little more specific.

Reply:
done.